arrow/arrow.py

```diff
@@ -79,8 +79,8 @@ def now(cls, tzinfo=None):
 
         '''
 
-        utc = datetime.utcnow().replace(tzinfo=dateutil_tz.tzutc())
-        dt = utc.astimezone(dateutil_tz.tzlocal() if tzinfo is None else tzinfo)
+        tzinfo = tzinfo or dateutil_tz.tzlocal()
```

Comment:
This changes the behaviour of `tzinfo`.  Before, tzlocal was assumed if it was `None`, now tzlocal is used for anything falsy.

Can you change to this (or something equivalent):

```python
if tzinfo is None:
  tzinfo = dateutil_tz.tzlocal()
```

Reply:
Fair enough I suppose.